cluster/saltbase/salt/kube-dns/skydns-rc.yaml.sed

```diff
@@ -75,6 +75,7 @@ spec:
         # command = "/kube-dns"
         - --domain=$DNS_DOMAIN.
         - --dns-port=10053
+        $FEDERATIONS_DOMAIN_MAP
```

Comment:
This might break the .sed consumers.

Also, should it be ``--federations=$FEDERATIONS_DOMAIN_MAP``` ?

We might want to leave this change out till we get some federation testing on systems that use the .sed files.